frontend/src/TooljetDatabase/TooljetDatabasePage/index.jsx

```diff
@@ -1,34 +1,71 @@
-import React, { useContext } from 'react';
+import React, { useContext, useState } from 'react';
 import cx from 'classnames';
 import Table from '../Table';
 import Sidebar from '../Sidebar';
 import { TooljetDatabaseContext } from '../index';
 import EmptyFoldersIllustration from '@assets/images/icons/no-queries-added.svg';
+import Warning from '../Icons/warning.svg';
+import { ButtonSolid } from '@/_ui/AppButton/AppButton';
 import { isEmpty } from 'lodash';
+import Plus from '@/_ui/Icon/solidIcons/Plus';
+import { tooljetDatabaseService } from '@/_services';
+import { toast } from 'react-hot-toast';
+import Drawer from '@/_ui/Drawer';
+import CreateTableForm from '../Forms/TableForm';
+import { BreadCrumbContext } from '@/App/App';
 
 const TooljetDatabasePage = ({ totalTables, collapseSidebar }) => {
-  const { selectedTable } = useContext(TooljetDatabaseContext);
-
+  const { organizationId, setSelectedTable, setTables, selectedTable } = useContext(TooljetDatabaseContext);
+  const [isCreateTableDrawerOpen, setIsCreateTableDrawerOpen] = useState(false);
```

Comment:
The drawer transition doesn't happen because this state is not stored in `EmptyState` component. So whenever the state changes, this component get re-rendered.